api/api/src/test/scala/uk/ac/wellcome/platform/api/services/WorksServiceTest.scala

```diff
@@ -269,6 +269,49 @@ class WorksServiceTest
       )
     }
 
+    it("Should simple_query_syntax into account and not fail when it's invalid") {
+      val workEmu = createIdentifiedWorkWith(
+        title = "a b c"
+      )
+
+      // This would land up with a too_many_clauses error due to the precedence syntax
+      assertSearchResultIsCorrect(
+        query = "(a b c d e) h"
+      )(
+        allWorks = List(workEmu),
+        expectedWorks = List(workEmu),
+        expectedTotalResults = 1
+      )
+    }
+
+    it("Should work with simple_query_syntax PHRASE syntax (\"term\")") {
```

Comment:
Including an example of the syntax is much more useful!

```suggestion
    it("matches results with the PHRASE syntax (\"term\")") {
```